src/plugins/visualizations/public/vis_types/vis_type_alias_registry.ts

```diff
@@ -86,8 +86,17 @@ export interface VisualizationsAppExtension {
 }
 
 export interface VisTypeAlias {
-  aliasPath: string;
-  aliasApp: string;
+  alias:
+    | {
+        app: string;
+        path: string;
+      }
+    /**
+     * Use this when your visualization uses inline editing and does not have a specific app
+     * to redirect to. This will default to the embeddable factory's editing method (`getExplicitInput`),
+     * which should handle the inline editing.
+     */
+    | { embeddableType: string };
```

Comment:
I chose this type so that, if we decide to allow **creation** of inline aliases (such as the Links embeddable) from the visualization listing page, the visualization wizard's `NewVisModal` component will need access to the embeddable type in order to fetch the correct factory.

Reply:
Why not simply provide a generic editing callback which opens the editing UI using whatever system the consumer decides to use (could be embeddable, could be something else)?